src/pages/singleDirection.tsx

```diff
@@ -82,11 +88,12 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
     }
     const operatorCookie = cookies[OPERATOR_COOKIE];
     const serviceCookie = cookies[SERVICE_COOKIE];
-    const fareTypeCookie = cookies[FARE_TYPE_COOKIE];
     const passengerTypeCookie = cookies[PASSENGER_TYPE_COOKIE];
     const nocCode = getNocFromIdToken(ctx);
 
-    if (!operatorCookie || !serviceCookie || !fareTypeCookie || !passengerTypeCookie || !nocCode) {
+    const fareTypeAttribute = getSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE);
```

Comment:
Looks like we could remove fareTypeAttribute from here too, as it isn't used in the API?

Reply:
good spot